src/main/java/com/google/sps/data/Charity.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps.data;
```

Comment:
We changed the file location for servlet to  src/main/java/com/googlesps/feedon/servlets/